langchain/src/vectorstores/tests/clickhouse.int.test.ts

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable no-process-env */
+import { test, expect } from "@jest/globals";
+
+import { ClickHouseStore } from "../clickhouse.js";
+// Import OpenAIEmbeddings if you have a valid OpenAI API key
+import { HuggingFaceInferenceEmbeddings } from "../../embeddings/hf.js";
+import { Document } from "../../document.js";
+
+test.skip("ClickHouseStore.fromText", async () => {
```

Comment:
!!!

Reply:
is skip intended?

Reply:
It's the integration test and will need a ClickHouse account to run them. We checked other similar int tests and found they all have "skip" so we also added it.